package.json

```diff
@@ -22,7 +22,8 @@
     "build": "yarn build:es5 && yarn test && yarn build:min"
   },
   "dependencies": {
-    "base-x-bytearray": "^0.1.2",
+    "base-x": "^3.0.0",
+    "buffer": "^5.0.6",
```

Comment:
@zachferland it's used in base-x. I experimented a with specifying it or not and it seemed to change the output. But looking at it, it doesn't seem to be using the 5.0.6  version anyway